assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { noop } from 'lodash';
+import { format, parseISO } from 'date-fns';
+
+import Banner from '@common/Banners/Banner';
+import Button from '@common/Button';
+import Modal from '@common/Modal';
+import PageHeader from '@common/PageHeader';
+import Table from '@common/Table';
+import Tooltip from '@common/Tooltip';
+
+const defaultUsers = [];
+
+function Users({
+  onDeleteUser = noop,
+  navigate = noop,
+  users = defaultUsers,
+  loading = true,
+}) {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [user, setUser] = useState(null);
+
+  const usersTableConfig = {
+    pagination: true,
+    usePadding: false,
+    columns: [
+      {
+        title: 'Username',
+        key: 'username',
+        render: (content, item) => (
+          <Link
+            className="text-jungle-green-500 hover:opacity-75"
+            to={`/users/${item.id}/edit`}
+          >
+            {content}
+          </Link>
+        ),
+      },
+      {
+        title: 'Full Name',
+        key: 'fullname',
+      },
+      {
+        title: 'Email',
+        key: 'email',
+      },
+      {
+        title: 'Status',
+        key: 'enabled',
+        render: (content, item) => (
+          <span>{item.enabled ? 'Enabled' : 'Disabled'}</span>
+        ),
+      },
+      {
+        title: 'Created',
+        key: 'created_at',
+        render: (content, item) => (
+          <span>{format(parseISO(item.created_at), 'MMMM dd, yyyy')}</span>
+        ),
+      },
+      {
+        title: 'Actions',
+        key: 'actions',
+        render: (content, item) => (
+          <Button
+            className="text-red-500 text-left w-auto"
+            size="small"
+            type="transparent"
+            disabled={item.id === 1}
+            onClick={() => {
+              setModalOpen(true);
+              setUser(item);
+            }}
+          >
+            <Tooltip
+              content="Admin user can not be deleted"
+              isEnabled={item.id === 1}
+            >
+              Delete
+            </Tooltip>
+          </Button>
+        ),
+      },
+    ],
+  };
+
+  return (
+    <div className="flex flex-wrap">
+      <div className="flex w-1/2 h-auto overflow-hidden overflow-ellipsis break-words">
+        <PageHeader className="font-bold">Users</PageHeader>
+      </div>
+      <div className="flex w-1/2 justify-end">
+        <div className="flex w-fit whitespace-nowrap">
+          <Button
+            className="inline-block mx-1 border-green-500 border"
```

Comment:
@EMaksy Following up on our conversation on Friday, I realised that the `border-green-500` and `border` classes can be removed here since they clearly visible when the button is disabled in the Loading state.